### ISSUE TITLE
add command line arguments to train-loglinear which specify (hyper)parameters

### ISSUE THREAD

**Issue by wammar** [ldmt-muri/alignment-with-openfst#25]
in particular, specify:
-  proposal distribution
-  learning rate
-  stopping criteria
-  regularization type and strength
-  batch size
-  optimization algorithm
